build.cmd

```diff
@@ -26,26 +19,9 @@ setlocal enabledelayedexpansion
   call "%~dp0.\dotnet-install.cmd" || exit /b 1
 
   set procedures=
-
-  if not defined lv_api_only (
-    set procedures=!procedures! build_procdomain
```

Comment:
Before removing all of this, I think we need two things:

1. Update the README to cover how the harness works.
2. Write and reference an issue from this PR that describes what's happening and why.

It's fine to include this change in this PR, but we just need to do these things before merging.